### PR TITLE
Jon Ng Instagram Challenge

### PR DESCRIPTION
**Pull request by jonathanngkh**

### CODE REVIEW

File: spec/features/likes_feature_spec.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  scenario 'liking a picture twice unlikes it' do
+    visit '/pictures'
+    click_link 'Like'
+    expect(page).to have_link 'Unlike'
+    click_link 'Unlike'
+    expect(page).to have_content '0 likes'
+    expect(Picture.all[0].likes.count).to eq 0
+  end
+
+  def sign_up
```
Comment (dwatson62) [inline spec/features/likes_feature_spec.rb]: Can extract this into a helper.rb file, so you don't have to declare it each time

File: spec/features/likes_feature_spec.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  scenario 'liking a picture' do
+    visit '/pictures'
+    click_link 'Like'
+    expect(page).to have_content '1 like'
+    expect(Picture.all[0].likes.count).to eq 1
+    expect(Picture.all[0].likes[0].user_id).not_to eq nil
+  end
+
+  scenario 'liking a picture twice unlikes it' do
```
Comment (dwatson62) [inline spec/features/likes_feature_spec.rb]: Can use expect { click_link 'Like' }.to change { Like.count }.by 1 
More precise

File: spec/features/comments_feature_spec.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+feature 'comments' do
+  before do
+    sign_up
```
Comment (ptolemybarnes) [inline spec/features/comments_feature_spec.rb]: This method is very opaque. Who am I signing up as?

File: spec/features/pictures_feature_spec.rb [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    end
+  end
+
+  context 'Adding pictures' do
+    before  do
+      sign_up
+    end
+
+    scenario 'form to fill in' do
+      visit '/pictures'
```
Comment (ptolemybarnes) [inline spec/features/pictures_feature_spec.rb]: It's better to use path helpers than hand write the string.

**Comment by ptolemybarnes** [comment]
:checkered_flag:

**Pull request closed without merging**